tests/rebuild_test.go

```diff
@@ -28,7 +28,7 @@ func TestRebuild_InstallNewBundle(t *testing.T) {
 
 	// Install a bundle without building first
 	installOpts := porter.InstallOptions{}
-	err = installOpts.Validate([]string{}, p.Context)
+	err = installOpts.Validate([]string{}, p.CNAB, p.Context)
```

Comment:
Yeah seeing adding this extra arg everywhere, if we change it to passing `p` instead, I think that would help in the future to avoid cascading changes like this. What do you think?

Reply:
Good call; simplified to just passing `p`.